loop_rate: 3
map_frame: "map"
base_frame: "base_link"
step: 0.5
distance_to_wall: 1.0
inflation: 0.11
angle: -1.57
timeout: 9.0
file_path: "wall_inspection.data" # persisted state file path. Keep it empty to start from scratch
has_cam: false
robot_radius: 0.3
